generators/pkg/writer/write.go

```diff
@@ -0,0 +1,20 @@
+// Package writer provides a method to write the generated task
+// to the yaml document.
+package writer
+
+import (
+	"log"
+
+	v1beta1 "github.com/tektoncd/pipeline/pkg/apis/pipeline/v1beta1"
+	"sigs.k8s.io/yaml"
```

Comment:
Nice find! Perhaps we should standardize on this for reading as well. (if we do this, let's make the change in another PR).

Reply:
Do you mean the `parser`? Maybe change it to use this package as well?